sdk/monitor/azure-monitor-query/samples/champion_scenarios.md

```diff
@@ -0,0 +1,288 @@
+import os
+import pandas as pd
+from azure.monitor.query import LogsQueryClient, LogsBatchQuery
+from azure.identity import DefaultAzureCredential
+
+
+credential  = DefaultAzureCredential()
+
+client = LogsQueryClient(credential)
+
+requests = [
+    LogsBatchQuery(
+        query="AzureActivity | summarize count()",
+        timespan=timedelta(hours=1),
+        workspace_id= os.environ['LOG_WORKSPACE_ID']
+    ),
+    LogsBatchQuery(
+        query= """AppRequests | take 5  |
+            summarize avgRequestDuration=avg(DurationMs) by bin(TimeGenerated, 10m), _ResourceId""",
+        timespan=(datetime(2021, 6, 2), timedelta(hours=1)),
+        workspace_id= os.environ['LOG_WORKSPACE_ID']
+    ),
+    LogsBatchQuery(
+        query= """AppRequests | take 5  |
+            summarize avgRequestDuration=avg(DurationMs) by bin(TimeGenerated, 10m), _ResourceId""",
+        workspace_id= os.environ['LOG_WORKSPACE_ID'],
+        timespan=(datetime(2021, 6, 2), datetime(2021, 6, 3)),
+        include_statistics=True
+    ),
+]
+results = client.query_batch(requests)
+
+for response in results:
+    if response.error is not None:
+        error = response.error.innererror
+        print(error)
+    
+    table = response.tables[0]
+    df = pd.DataFrame(table.rows, columns=table.columns)
+    print(df)
+    print("\n\n-------------------------\n\n")
+
+"""
+   count_
+0       2
+
+
+-------------------------
+
+
+              TimeGenerated                                        _ResourceId  avgRequestDuration
+0 2021-06-02 00:20:00+00:00  /subscriptions/<subscription id>...            18.12380
+1 2021-06-02 00:00:00+00:00  /subscriptions/<subscription id>...            20.84805
+2 2021-06-02 00:10:00+00:00  /subscriptions/<subscription id>...            19.72410
+3 2021-06-02 00:30:00+00:00  /subscriptions/<subscription id>...            19.41265
+4 2021-06-02 00:40:00+00:00  /subscriptions/<subscription id>...            19.17145
+
+
+-------------------------
+
+
+
+              TimeGenerated                                        _ResourceId  avgRequestDuration
+0 2021-06-02 00:20:00+00:00  /subscriptions/<subscription id>...            18.12380
+1 2021-06-02 00:00:00+00:00  /subscriptions/<subscription id>...            20.84805
+2 2021-06-02 00:10:00+00:00  /subscriptions/<subscription id>...            19.72410
+3 2021-06-02 00:30:00+00:00  /subscriptions/<subscription id>...            19.41265
+4 2021-06-02 00:40:00+00:00  /subscriptions/<subscription id>...            19.17145
+
+
+
+-------------------------
+"""
+```
+
+#### Results in Key Value form
+
+
+Very Simlar to above:
+
+```python
+for response in results:
+    if response.error is not None:
+        error = response.error.innererror
+        print(error)
+    
+    table = response.tables[0]
+    df = pd.DataFrame(table.rows, columns=table.columns)
+    print(df.to_dict(orient='records'))
+    print("\n\n-------------------------\n\n")
+```
```

Comment:
you don't have the results here for this one. I'm assuming it's because the results are the same as the previous scenario? If so, it might be useful to have a comment saying something like:
`### output is the same as "...." scenario